i18n/en.json

```diff
@@ -48,6 +51,7 @@
 	"block-log": "Block log",
 	"block-longest": "Longest block",
 	"blocks": "Blocks",
+	"blocks-desc": "$2 {{PLURAL:$1|block|blocks}} administered",
```

Comment:
Just curious, why do we include this in the score? I thought Admin Score was more for non-admins. Even for admins, it seems odd to only account for blocks and not other admin actions.

Reply:
I'm not quite sure, our AdminScore was copied from the old one entirely.  I would be OK removing it, maybe we should open a Phabricator task to discuss it?

Reply:
Nah I think it's reasonable to just remove it. Doesn't have to be part of this PR, though!